app/static/src/app/components/header/AppHeader.vue

```diff
@@ -18,7 +18,7 @@
         <li><router-link id="all-sessions-link" class="dropdown-item" to="/sessions">All Sessions</router-link></li>
       </ul>
     </span>
-    <span class="navbar-version navbar-text">WODIN v{{ wodinVersion }}</span>
+      <version-menu :wodin-version="wodinVersion"></version-menu>
```

Comment:
trivial, but looks like indent is off here?

```suggestion
    <version-menu :wodin-version="wodinVersion"></version-menu>
```